tests/tests_geomstats/test_visualization_stiefel_manifold.py

```diff
@@ -0,0 +1,149 @@
+"""Stiefel manifold visualization test module."""
+# import geomstats.backend as gs
+import matplotlib
+import matplotlib.pyplot as plt
+import numpy as np
+
+import tests.conftest
+from geomstats.geometry.stiefel import Stiefel
+from geomstats.visualization.stiefel_manifold import (
+    Arrow2D,
+    StiefelCircle,
+    StiefelSphere,
+)
+
+matplotlib.use("Agg")  # NOQA
+
+
+class TestStiefelManifold(tests.conftest.TestCase):
+    """Class to test all the functions for S(2,2)."""
+
+    def setup_method(self):
+        """Initialize variables."""
+        self.n_samples = 10
+        self.v21 = Stiefel(2, 1)
+        self.v22 = Stiefel(2, 2)
+        self.v31 = Stiefel(3, 1)
+        self.St_sph = StiefelSphere()
+        self.St_cir = StiefelCircle()
+
+        self.sph_ax = self.St_sph.set_ax(ax=None)
+        self.cir_ax = self.St_cir.set_ax(ax=None)
+
+        plt.figure()
+
+    def test_stiefel_sphere_set_ax(self):
+        """Set ax for sphere."""
+        self.St_sph.set_ax(ax=None)
+
+    def test_stiefel_sphere_set_view(self):
+        """Set view for sphere."""
+        self.St_sph.set_view()
+
+    def test_stiefel_sphere_draw(self):
+        """Draw a stiefel manifold."""
+        self.St_sph.draw()
+
+    def test_stiefel_sphere_coordinates_transformation(self):
+        """Makes random points from distribution."""
+        points = self.v31.random_uniform(10)[2, :, 0]
+        self.St_sph.coordinates_transformation(points)
+
+    def test_stiefel_sphere_add_points(self):
+        """Take, add points from random distribution on manifold, visualize."""
+        points = self.v31.random_uniform(10)
+        self.St_sph.add_points(points)
+        self.St_sph.draw_points(self.sph_ax)
+
+    def test_stiefel_sphere_draw_points(self):
+        """Take points from random distribution on manifold and visualize."""
+        points = self.v31.random_uniform(10)
+        self.St_sph.draw_points(points)
+
+    def test_stiefel_sphere_clear_points(self):
+        """Clear points, draw points to verify points were cleared."""
+        self.St_sph.clear_points()
+        self.St_sph.draw_points(self.sph_ax)
+
+    def test_stiefel_sphere_draw_mesh(self):
```

Comment:
Put this line above the test functions that fail in pytorch:
```@geomstats.tests.np_only```

after importing `geomstats.tests` at the top of the file.

The github actions errors will go away